csharp.benchmark/Program.cs

```diff
@@ -44,9 +34,7 @@ public static int Main()
                 }
 
                 return 0;
-            }
-
-            catch (Exception exception)
+            } catch (Exception exception)
```

Comment:
Same remark as above. The previous style was more readable (ending brace and catch on a separate line).